settings.json

```diff
@@ -15,28 +15,32 @@
       "description": "Load Data",
       "helperText": "Load preprocessed data by pressing the button",
       "name" : "loadData",
-      "readOnly": "true"
+      "readOnly": "true",
+      "imageLink": "/instructions/loadData.png"
```

Comment:
I just realized, these likely wont load correctly on GH pages, since we are not at the root.